sled-agent/src/bootstrap/early_networking.rs

```diff
@@ -552,9 +569,9 @@ fn retry_policy_switch_mapping() -> ExponentialBackoff {
 /// [`super::params::RackInitializeRequest`] necessary for use beyond RSS. This
 /// is just for the initial rack configuration and cold boot purposes. Updates
 /// will come from Nexus in the future.
-#[derive(Clone, Debug, Deserialize, Serialize)]
+#[derive(Clone, Debug, Deserialize, Serialize, JsonSchema)]
 pub struct EarlyNetworkConfig {
-    // The version of data. Always `1` when created from RSS.
```

Comment:
I found it confusing given that the config could be created by RSS, and then it could get modified by something resulting in a generation number > 1. That does not change the fact that it was created by RSS, it's just been altered a bit. It could even be changed back to exactly what RSS created with a version number > 1.

Reply:
I changed the comments here to reflect reality more.